ui/page/file/view.jsx

```diff
@@ -120,7 +123,8 @@ function FilePage(props: Props) {
         <div className="file-page__secondary-content">
           <div>
             {RENDER_MODES.FLOATING_MODES.includes(renderMode) && <FileTitle uri={uri} />}
-            <CommentsList uri={uri} linkedComment={linkedComment} />
+            {disableComments && <Empty text={__('Comments are disabled here.')} />}
```

Comment:
I think this should say that the owner disabled comments, not that they are just disabled.

Maybe "The creator of this content has disabled comments."